core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2453,7 +2448,7 @@ public String getUrlChildPrefix() {
     @CheckForNull
     public String getConfiguredRootUrl() {
         JenkinsLocationConfiguration config = JenkinsLocationConfiguration.get();
-        return config != null ? config.getUrl() : null;
+        return config.getUrl();
```

Comment:
This can never be null because `ExtensionList#getInstance` throws `IllegalStateException` if the class cannot be found.